### PR TITLE
add exe 14 files

### PR DESCRIPTION
**Pull request by norona-marynicole**
add exe 14 files

### CODE REVIEW

File: norona_marynicole/index.html [hunk 1]
```diff
@@ -43,6 +43,8 @@ <h2 class="goals">MY GOALS</h2>
         <li>To be rich and successful in life</li>
       </ol>
 
+      <a href="./countries.html" id="countries-pages">Countries</a>
```
Comment (MentokAlmera) [inline norona_marynicole/index.html]: can you make it visible cause at first glance it's hard for me to find it.

File: norona_marynicole/countries.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    <link rel="stylesheet" href="countries.css" />
+  </head>
+  <body>
+    <div class="container">
+      <h1>Countries of the World</h1>
+      <input
+        type="text"
+        id="countryInput"
+        placeholder="Enter country name..."
+      />
```
Comment (MentokAlmera) [inline norona_marynicole/countries.html]: Remove unnecessary breaking of lines

https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit?fbclid=IwAR2mb5Qre33CFfRuTtpvAN1dTOpbICXFQJSlA9Pa6x8KcjHwHelDlK8iGzQ#slide=id.g29e6ce31160_0_27

File: norona_marynicole/countries.css [hunk 3]
```diff
@@ -0,0 +1,59 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    background-color: #ffcccc;
+}
+
```
Comment (MentokAlmera) [inline norona_marynicole/countries.css:10]: remove extra lines

File: norona_marynicole/countries.css [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+.container {
+    max-width: 600px;
+    margin: 0 auto;
+    padding: 20px;
+    background-color: #fff;
+    border-radius: 8px;
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+}
+
```
Comment (MentokAlmera) [inline norona_marynicole/countries.css:19]: remove extra lines

File: norona_marynicole/countries.css [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+h1 {
+    text-align: center;
+    margin-bottom: 20px;
+}
+
```
Comment (MentokAlmera) [inline norona_marynicole/countries.css:24]: remove extra line

File: norona_marynicole/countries.css [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+#countryInput {
+    width: calc(100% - 80px);
+    padding: 10px;
+    font-size: 16px;
+    border: 1px solid #e86f6f;
+    border-radius: 4px;
+}
+
```
Comment (MentokAlmera) [inline norona_marynicole/countries.css:32]: remove extra line

File: norona_marynicole/countries.css [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+    color: #fff;
+    border: none;
+    border-radius: 4px;
+    cursor: pointer;
+}
+
+button:hover {
+    background-color: #d481b7;
+}
+
```
Comment (MentokAlmera) [inline norona_marynicole/countries.css:47]: remove extra line

File: norona_marynicole/countries.js [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+      }
+      return response.json();
+    })
+    .then(function (countryData) {
+      let country = countryData[0];
+      let details = `
+            <h2>Country Details - ${country.name.common}</h2>
+            <img src="${country.flags.svg}" alt="Flag of ${
+        country.name.common
+      }" width="100">
```
Comment (MentokAlmera) [inline norona_marynicole/countries.js]: I suggest
Flag of 
              ${country.name.common}

**Review by MentokAlmera — CHANGES_REQUESTED** [review]
Please review it. Thank you.

**Review by MentokAlmera — APPROVED** [review]
Great job! It's already working <3

**Comment by norona-marynicole** [comment]
Thank you so much po.

**Pull request merged**